src/src/smartAudio.c

```diff
@@ -251,26 +264,22 @@ void smartaudioProcessModePacket(void)
     // Set PIR and POR. POR is no longer used in SA2.1 and is treated like PIR
     myEEPROM.pitmodeInRange = bitRead(data, 0);
     myEEPROM.pitmodeOutRange = bitRead(data, 1);
-    uint8_t clearPitMode = bitRead(data, 2);
     myEEPROM.unlocked = bitRead(data, 3);
 
     // This bit is only for CLEARING pitmode.
-    if (clearPitMode) {
+    if (bitRead(data, 2)) {
```

Comment:
I like keeping clearPitMode for readability, but Im easy either way.

Reply:
change reverted back to original :)